monitor/api/models/schema.py

```diff
@@ -1,23 +1,41 @@
-from typing import List, Optional
+from typing import List, Literal, Optional
 from pydantic import BaseModel, validator
 
 
-class ModelSchema(BaseModel):
+class ArtifactSchema(BaseModel):
     name: str
     unique_id: str
-    database_name: str
-    schema_name: str
-    table_name: str
     owners: Optional[str]
     tags: Optional[str]
-    package_name: str
+    package_name: Optional[str]
     description: Optional[str]
     full_path: str
 
 
-class NormalizedModelSchema(ModelSchema):
+class ModelSchema(ArtifactSchema):
+    database_name: str
+    schema_name: str
+    table_name: str
+
+
+class SourceSchema(ArtifactSchema):
+    database_name: str
+    schema_name: str
+    table_name: str
+
+
+class ExposureSchema(ArtifactSchema):
+    url: Optional[str]
+    type: Literal["dashboard", "notebook", "analysis", "ml", "application"]
```

Comment:
Are you sure type and owner_email are not optional?

Reply:
100%